### ISSUE TITLE
docker-compose build is not working

### ISSUE THREAD

**Issue by nibro7778** [dotnet/eShopOnContainers#1824]
I am trying to set up a project and getting the following issue. Also, I am not good at npm and node :( looks like some versioning issue. I tried by upgrading the node version to `17.3.1` (https://nodejs.org/en/download/releases/) but the build failed with some other error at `webspa` service

![image](https://user-images.githubusercontent.com/16951150/149766145-7f5f9896-f7d0-4377-9d29-bd9788b7e204.png)

Am I missing something?

**Comment by nishanil** [comment]
Hey @nibro7778 I just verified this. It is working as expected for me. It is probably an interim error. Can you try again deleting the images?

**Comment by nibro7778** [comment]
Hey @nishanil, 

I tried deleting all images and containers. Also, tried by `docker system prune` but still I am unable to run it from the `main` branch. I tried updating the node to `12.0` (`dev` branch) but it throws the following error.

![image](https://user-images.githubusercontent.com/16951150/150666696-3f8ac64e-4a76-47a4-915f-8a1268bcf62c.png)

What I found is Setup working properly with the `dev` branch but not with the `main` branch.

Able to run `docker-compose build` and `docker-compose up` on `dev` branch.
